src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -73,7 +73,7 @@ private static Task BuildTocMap(Document fileToBuild, TableOfContentsMapBuilder
             var errors = new List<DocfxException>();
             var referencedDocuments = new List<Document>();
             var referencedTocs = new List<Document>();
-            var tocViewModel = TableOfContentsParser.Load(
+            var (tocViewModel, loadErros) = TableOfContentsParser.Load(
```

Comment:
>loadErros [](start = 31, length = 9)

errors?

Reply:
loadErrors, typo

---
In reply to: [197333863](https://github.com/dotnet/docfx/pull/2918#discussion_r197333863) [](ancestors = 197333863)